tests/util/workspace.rs

```diff
@@ -0,0 +1,17 @@
+mod util;
+
+/// Test utility to set up test dura environment
+pub struct Workspace {
```

Comment:
Doesn't look like you're using this. I used to have a different form of this, but removed it because it's not flexible enough (I needed multiple Git repos)